lua/lvim/keymappings.lua

```diff
@@ -24,13 +24,27 @@ local mode_adapters = {
 -- Append key mappings to lunarvim's defaults for a given mode
 -- @param keymaps The table of key mappings containing a list per mode (normal_mode, insert_mode, ..)
 function M.append_to_defaults(keymaps)
+  local default = M.get_defaults()
+  lvim.keys = lvim.keys or default
   for mode, mappings in pairs(keymaps) do
-    for k, v in ipairs(mappings) do
+    lvim.keys[mode] = lvim.keys[mode] or default[mode]
+    for k, v in pairs(mappings) do
       lvim.keys[mode][k] = v
     end
   end
 end
 
+-- Unsets all keybindings defined in keymaps
+-- @param keymaps The table of key mappings containing a list per mode (normal_mode, insert_mode, ..)
+function M.clear(keymaps)
+  for mode, mappings in pairs(keymaps) do
+    mode = mode_adapters[mode] and mode_adapters[mode] or mode
+    for key, _ in pairs(mappings) do
+      pcall(vim.api.nvim_del_keymap, mode, key)
```

Comment:
Actually, that could be solved here, we can just compare the values and only delete it if it's the default one.

Reply:
This only clears what has been set in `lvim.keys` not _all_ keybindings, so that's more or less the current behavior.